tmt/steps/provision/podman.py

```diff
@@ -126,20 +133,24 @@ class GuestContainer(tmt.Guest):
     container: Optional[str]
     user: str
     force_pull: bool
+    parent: tmt.steps.Step
 
-    def wake(self):
+    def wake(self) -> None:
         """ Wake up the guest """
         self.debug(
             f"Waking up container '{self.container}'.", level=2, shift=0)
 
-    def start(self):
+    def start(self) -> None:
         """ Start provisioned guest """
         if self.opt('dry'):
             return
         # Check if the image is available
-        image_id = self.podman(
+        assert self.image is not None
+        podman_output = self.podman(
             ['images', '-q', self.image],
-            message=f"Check for container image '{self.image}'.")[0].strip()
+            message=f"Check for container image '{self.image}'.")
+        assert podman_output[0] is not None
```

Comment:
Can we rather return an error here, if the output is nothing, we should raise a resonable error ... ?

Reply:
Okay, changed it to raising a `tmt.utils.RunError`.